### PR TITLE
Feature/php8

### PR DESCRIPTION
**Pull request by Chris53897**
Hi. This is the idea to create a new major release 2.0 with only php >= 8 and symfony 4.4.31 || ^5.3 || ^6.0 support.
This bundle looks stable and low maintained. With this relase it will be easier to maintain and to work with symfony 6.
[See Testrun](https://github.com/Chris53897/KnpMarkdownBundle/actions/runs/1298971561)

Please let me know what you think about this idea.

### CODE REVIEW

File: Parser/ParserManager.php [hunk 1]
```diff
@@ -9,28 +9,19 @@ class ParserManager
     /**
      * @var MarkdownParserInterface[]
      */
-    private $parsers = array();
+    private array $parsers = array();
```
Comment (elkuku) [inline Parser/ParserManager.php:12]: You may use short array syntax here too ;)

Comment (Chris53897) [reply Parser/ParserManager.php:12]: I will take a look on this after the maintainers will give some feedback about this PR ;)

**Comment by elkuku** [comment]
I :heart: this!

File: Parser/ParserManager.php (hunk 1 above)
Comment (elkuku) [reply Parser/ParserManager.php:12]: Yeah I was just spotting it..

I hope some PR that brings in SF6 support will be merged some day soon :(

**Comment by Chris53897** [comment]
Closed because of https://github.com/KnpLabs/KnpMarkdownBundle/issues/116

**Pull request closed without merging**